community-operators/mongodb/mongodb-enterprise.v0.3.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,316 @@
+        x-descriptors:
+        - 'urn:alm:descriptor:io.kubernetes:core:v1:ConfigMap'
+
+      - description: MongoDB version to be installed.
+        displayName: Version
+        path: version
+        x-descriptors:
+        - 'urn:alm:descriptor:com.tectonic.ui:label'
+
+      - description: How many Config Servers will be deployed
+        displayName: Config Server Count
+        path: configServerCount
+        x-descriptors:
+        - 'urn:alm:descriptor:com.tectonic.ui:label'
+
+      - description: How many MongoDB Servers per Shard will be deployed
+        displayName: MongoDB Servers per Shard
+        path: mongodsPerShardCount
+        x-descriptors:
+        - 'urn:alm:descriptor:com.tectonic.ui:label'
+
+      - description: How many mongos will be deployed
+        displayName: Mongos (MongoDB Shard) to be deployed
+        path: mongosCount
+        x-descriptors:
+        - 'urn:alm:descriptor:com.tectonic.ui:label'
+
+      - description: Amount of Shards to be deployed
+        displayName: Shards
+        path: shardCount
+        x-descriptors:
+        - 'urn:alm:descriptor:com.tectonic.ui:label'
+
+      - description: The service where this MongoDB deployment will be listening.
+        displayName: Service
+        path: service
+        x-descriptors:
+        - 'urn:alm:descriptor:io.kubernetes:Service'
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
+  install:
+    strategy: deployment
+    spec:
+      deployments:
+      - name: mongodb-enterprise-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              k8s-app: mongodb-enterprise-operator
+          template:
+            metadata:
+              labels:
+                k8s-app: mongodb-enterprise-operator
+            spec:
+              serviceAccountName: mongodb-enterprise-operator
+              imagePullSecrets:
+                - name: ''
+
+              containers:
+              - name: mongodb-enterprise-operator
+                image: 'quay.io/mongodb/mongodb-enterprise-operator:0.7'
+                imagePullPolicy: Always
+                resources:
+                  limits:
+                    cpu: 200m
+                    memory: 100Mi
+                  requests:
+                    cpu: 100m
+                    memory: 50Mi
+
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: MANAGED_SECURITY_CONTEXT
+                  value: 'true'
+                - name: OPERATOR_ENV
+                  value: prod
+                - name: MONGODB_ENTERPRISE_DATABASE_IMAGE
+                  value: registry.connect.redhat.com/mongodb/enterprise-database
```

Comment:
This is the same image as for the certified version of the operator. I'm not sure what other differences there are between this operator and the certified version; they look the same to me right down to the CSV version.